### ISSUE TITLE
Publish python wheels for apple silicon Macs

### ISSUE THREAD

**Issue by willwade** [kxxt/aspeak#81]
so heres the code

`from aspeak import SpeechService, AudioFormat`

and it leads to

```
Traceback (most recent call last):
  File "/Users/willwade/GitHub/TranslateAndTTS/translatepb.py", line 9, in <module>
    from aspeak import SpeechService, AudioFormat
ImportError: cannot import name 'SpeechService' from 'aspeak' (/Users/willwade/.pyenv/versions/3.11.4/lib/python3.11/site-packages/aspeak/__init__.py)
```

I've tried py 3.8, 3.10.11, 3.9 and you can see above 3.11
I'm on a Mac M2

I have a feeling its the mac causing this issue. On Windows running py 3.8 its fine

**Comment by kxxt** [comment]
> I'm on a Mac M2
> I have a feeling its the mac causing this issue. 

Yes, your feeling is right. I haven't published wheels for apple silicon Macs. 
This could be fixed but I am busy right now and don't have time to do so.

For now, You can [build a wheel by yourself](https://github.com/kxxt/aspeak#python-wheel), or just use [the closed-source speech SDK](https://pypi.org/project/azure-cognitiveservices-speech/) from Microsoft

**Comment by kxxt** [comment]
Fixed in 6.1.0
